torch/csrc/cudnn/Types.cpp

```diff
@@ -31,4 +31,15 @@ PyObject * getTensorClass(PyObject *args)
   return NULL;
 }
 
+void THVoidTensor_assertContiguous(THVoidTensor *tensor)
+{
+  // Contiguity check
+  long long expectedStride = 1;
+  for (int i = tensor->nDimension-1; i >= 0; --i) {
+    if (tensor->stride[i] != expectedStride)
+      throw std::invalid_argument("cuDNN convolutions require contiguous weights");
```

Comment:
this error message is wrong. its used in many other places. would be nice to know which attribute name was non-contiguous when this error happens